server/sonar-server/src/main/java/org/sonar/server/setting/ws/EncryptAction.java

```diff
@@ -60,7 +59,7 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(SYSTEM_ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not 'Administer System' permission... or is that the name we give to root?